ts/output/chtml/FontData.ts

```diff
@@ -436,7 +411,11 @@ export type CssMap = {[name: number]: number};
 export function AddCSS(font: CHTMLCharMap, options: CharOptionsMap): CHTMLCharMap {
   for (const c of Object.keys(options)) {
     const n = parseInt(c);
-    Object.assign(FontData.charOptions(font, n), options[n]);
+    const data = options[n];
+    if (data.c) {
+      data.c = data.c.replace(/\\[0-9A-F]+/ig, (x) => String.fromCodePoint(parseInt(x.substr(1), 16)));
+    }
```

Comment:
These three lines convert from the current `content` string like `\338` for U+0338 to an actual unicode string.  This is to avoid having to change all the font data files.  The font update for CHTML will resolve that problem and will remove these lines.